functions/onprem/orborus/orborus.go

```diff
@@ -1150,6 +1152,18 @@ func deployWorker(image string, identifier string, env []string, executionReques
 		},
 		Resources: container.Resources{},
 	}
+    
+    _, err := os.ReadDir(certPath)
```

Comment:
Does it fail if certPath is an empty string? Or just err like we're used to? Can't we just do this inside an if statement that is only ran IF there is a string?

Reply:
yup doing it inside an if is what I'm going to do. But this was before I was setting some default value of `SHUFFLE_CERT_DIR` in the `shuffle.GetExternalClient` which is not a great solution. But yes there is not need because we will just be using a default path i.e `/certs` as we are mounting all the certificates from userspace to `/certs` in orborus.